src/app/dashboard/dashboard.component.ts

```diff
@@ -14,6 +14,6 @@ export class DashboardComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.view.body.subscribe(heroes => this.heroes = heroes);
+    this.view.body.subscribe((heroes: Heroes) => this.heroes = heroes);
```

Comment:
Why these changes? Applies to all components.
Generic signature is still automatic and linter is happy too. Unnescessary explicit code, used as an example.

Reply:
I had feeling that there was build or lint errors because of that, but it's certantly not the case now - would switch it back to the original.